source/options.tsx

```diff
@@ -8,18 +8,37 @@ import fitTextarea from 'fit-textarea';
 import {applyToLink} from 'shorten-repo-url';
 import * as indentTextarea from 'indent-textarea';
 
+import getTextNodes from './helpers/get-text-nodes';
 import {perDomainOptions} from './options-storage';
 import * as domFormatters from './github-helpers/dom-formatters';
 
+function parseSimpleInlineElement(element: Element, tag: string): void {
+	const splittingRegex = new RegExp(`<${tag}>(.+?)</${tag}>`, 'g');
+
+	for (const node of getTextNodes(element)) {
+		const fragment = new DocumentFragment();
+		for (const [index, text] of node.textContent!.split(splittingRegex).entries()) {
+			if (index % 2 && text.length >= 1) {
+				const createdElement = document.createElement(tag);
+				createdElement.textContent = text.trim();
+				fragment.append(createdElement);
+			} else if (text.length > 0) {
+				fragment.append(text);
+			}
+		}
+
+		if (fragment.children.length > 0) {
+			node.replaceWith(fragment);
+		}
+	}
+}
+
 function parseDescription(description: string): DocumentFragment {
 	const descriptionElement = <span>{description}</span>;
-	domFormatters.linkifyIssues(descriptionElement, {
-		baseUrl: 'https://github.com',
-		user: 'sindresorhus',
-		repository: 'refined-github'
-	});
 	domFormatters.linkifyURLs(descriptionElement);
 	domFormatters.parseBackticks(descriptionElement);
+	parseSimpleInlineElement(descriptionElement, 'i');
+	parseSimpleInlineElement(descriptionElement, 'kbd');
```

Comment:
This has been bugging me because we've ended up pars͡iǹģ H҉T͠M͚̼̱L̷ w̱̳i̓ͭͪ̅͌ṫ͓̼̬̜̔h͐͑͆̊̄͏̟ re̝̘̱̣̥͙͔̝̫͙̯̣ͣ͗ͨ̿̏̌ͤ̋́g̶̟̻̻̳̯̹̞̥̰̲͙̮̝͂̋̾͋̽̂̽̚e͋̔͆̑̌̎͗́̂ͮ̌̅͞҉̛̟͙̠͍̠͍̪͍̝̲̪̳͈̰͍x̷̡̡̢̖̗̞̫̠͉̪ͧ̄̎̌̎ͫ.

Here or in a separate PR I'd like to see the most lightweight Markdown parser being run directly to the description text to store HTML so the entire `parseDescription` function can be dropped

Reply:
Parser example: https://github.com/Holixus/nano-markdown
https://bundlephobia.com/result?p=nano-markdown@1.2.1

Reply:
> This has been bugging me because we've ended up pars͡iǹģ H҉T͠M͚̼̱L̷ w̱̳i̓ͭͪ̅͌ṫ͓̼̬̜̔h͐͑͆̊̄͏̟ re̝̘̱̣̥͙͔̝̫͙̯̣ͣ͗ͨ̿̏̌ͤ̋́g̶̟̻̻̳̯̹̞̥̰̲͙̮̝͂̋̾͋̽̂̽̚e͋̔͆̑̌̎͗́̂ͮ̌̅͞҉̛̟͙̠͍̠͍̪͍̝̲̪̳͈̰͍x̷̡̡̢̖̗̞̫̠͉̪ͧ̄̎̌̎ͫ.

😁 Yes I know, this is not the cleanest way, but I didn't want to completely change it to a real Markdown parser.

> Here or in a separate PR I'd like to see the most lightweight Markdown parser being run directly to the description text to store HTML so the entire `parseDescription` function can be dropped

I'll give it a shot in a new PR, I think this one is large enough 😅